helix-core/src/test/resources/log4j.properties

```diff
@@ -27,6 +27,7 @@ log4j.appender.R=org.apache.log4j.RollingFileAppender
 log4j.appender.R.layout=org.apache.log4j.PatternLayout
 log4j.appender.R.layout.ConversionPattern=%5p [%C:%M] (%F:%L) - %m%n
 log4j.appender.R.File=target/ClusterManagerLogs/log.txt
+log4j.appender.R.MaxBackupIndex=20
```

Comment:
What's the motivation for this change? Did we hit the limitation?
Once we have the test log cleaned up, we will have a smaller log and fit the default file size, right?

Reply:
removed.